docs/config.js

```diff
@@ -2,23 +2,21 @@ module.exports = {
   title: "Tendermint Core",
   description: "Documentation for Tendermint Core",
   dest: "./dist/docs",
-  base: "/",
+  base: "/docs/",
```

Comment:
Should we also remove this prefix from links in .md files?

Reply:
@faboweb I don't think that's necessary, right?

Reply:
can you point me to an example on the website?

Reply:
@melekes this `base: "/docs"` refers to the documentation website being run under a subdirectory of the Tendermint website. I do believe there are links that need to be fixed in the Tendermint docs to ensure they work on the docs website. But that probably belongs in a different PR (@jb @zramsay are the ones who would know more). This PR is a high priority merge for develop and master. @greg-szabo

Reply:
Then we should ask @ebuchman to include it into https://github.com/tendermint/tendermint/pull/2261

Reply:
Seems theres a few docs related things on develop that aren't on master. Can we get a separate PR against master that just copies these over, or at least whatever parts we need?

Reply:
Is https://github.com/tendermint/tendermint/pull/2336 enough?